constants.yml

```diff
@@ -114,6 +118,14 @@ company_status_detail:
     'petition-to-restore-dissolved' : "Petition to restore dissolved"
     'transformed-to-se' : "Transformed to SE"
     'converted-to-plc' : "Converted to PLC"
+overseas_acct_type:
+    'accounting-requirements-of-originating-country-apply' : "Accounting requirements of originating country apply"
+    'accounting_requirements_of_originating_country_do_not_apply': "Accounting requirements of originating country do not apply"
+overseas_acct_status:
```

Comment:
Do you mean, 'don't need to used shortened words here, just use overseas_account_status'